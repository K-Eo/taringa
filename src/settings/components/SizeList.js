import React from 'react'

import SizeButton from './SizeButton'
import { ITEM_BIG, ITEM_MEDIUM, ITEM_SMALL } from '../constants'

const SizeList = () => {
  return (
    <div className="form-inline">
      <SizeButton size={ITEM_BIG} icon="ta-items-big" />
      <SizeButton size={ITEM_MEDIUM} icon="ta-items-medium" />
      <SizeButton size={ITEM_SMALL} icon="ta-items-small" />
    </div>
  )
}

export default SizeList
