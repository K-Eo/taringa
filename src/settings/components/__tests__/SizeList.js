import React from 'react'

import SizeList from '../SizeList'

describe('Size List', () => {
  it('exists', () => {
    expect(SizeList).toBeDefined()
  })

  it('renders', () => {
    const wrapper = shallow(<SizeList />)

    expect(wrapper).toMatchSnapshot()
  })
})
