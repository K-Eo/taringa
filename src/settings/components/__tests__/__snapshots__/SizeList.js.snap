// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Size List renders 1`] = `
<div
  className="form-inline"
>
  <Connect(Button)
    icon="ta-items-big"
    size="settings/ITEM_BIG"
  />
  <Connect(Button)
    icon="ta-items-medium"
    size="settings/ITEM_MEDIUM"
  />
  <Connect(Button)
    icon="ta-items-small"
    size="settings/ITEM_SMALL"
  />
</div>
`;
